src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -106,31 +106,37 @@ export const TxDetails = ({ transaction, actions }: TxDetailsProps): ReactElemen
 
   return (
     <TxDetailsContainer>
-      <div className={cn('tx-summary', { 'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED' })}>
-        <TxSummary txDetails={data} />
-      </div>
-      <div
-        className={cn('tx-details', {
-          'no-padding': isMultiSendTxInfo(data.txInfo),
-          'not-executed': !data.executedAt,
-          'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED',
-        })}
-      >
-        <TxDataGroup txDetails={data} />
-      </div>
-      <div
-        className={cn('tx-owners', {
-          'no-owner': txLocation !== 'history' && !actions?.isUserAnOwner,
-          'will-be-replaced': transaction.txStatus === 'WILL_BE_REPLACED',
-        })}
-      >
-        <TxOwners txDetails={data} />
+      <div>
```

Comment:
Have you checked all forms of transactions after moving the order of these?

Reply:
I checked for all the transactions that we have in the task description. I think I've got them all but the ticket will be scrutinized by QA anyway. So far seems to work everywhere.